tests/test_checker/test_noqa.py

```diff
@@ -189,6 +189,7 @@
     'WPS519': 1,
     'WPS520': 1,
     'WPS521': 1,
+    'WPS522': 0,
```

Comment:
Please, include at least one error in `noqa.py`

Reply:
Done